Sources/tuist/TuistApp.swift

```diff
@@ -13,6 +13,10 @@ private enum TuistServer {
             try? ProcessEnv.setVar(Constants.EnvironmentVariables.verbose, value: "true")
         }
 
+        if CommandLine.arguments.contains("--quiet") {
+            try? ProcessEnv.setVar(Constants.EnvironmentVariables.quiet, value: "true")
+        }
+
```

Comment:
Can `--json` flag be used with `--quiet` or maybe they should be exclusive too? 

If we wanted to allow having both `--json` and `--quiet`, we couldn't add a `quite` case to logger type.

Reply:
They are exclusive – it doesn't make sense to use `--quiet` with `json`

Reply:
Alright, so I guess we can also throw an error if both `--json` and `--quiet` are passed, right?